api/v1/constants.go

```diff
@@ -193,3 +190,8 @@ const (
 	EnvVarCnbpEnabled                 = "COH_CNBP_ENABLED"
 	EnvVarCnbpLauncher                = "COH_CNBP_LAUNCHER"
 )
+
+var (
+	// The affinity topology key for fault domains.
```

Comment:
This is failing the code review step of the build. The comment needs to be in the format "AffinityTopologyKey ...", so something like `// AffinityTopologyKey is the affinity topology key for fault domains`

I know the comment is the same as previously but it now gets picked up for some reason.